### PR TITLE
Minor Kafka Update

### PR DESCRIPTION
**Pull request by fabianbaier**
Changelog:

- Making `/var/lib/kafka/data` the data directory ( https://github.com/kudobuilder/kudo/issues/90#issuecomment-461383910 )
- Changing deployment strategy to `serial`
- Renaming the Kafka instance from `small` to `kafka`
- Setting `BROKER_COUNT` from `3` to `1`

Background:

When giving demos on Minikube and preparing for the quickstart video retake ( https://github.com/kudobuilder/kudo/issues/108 ) I noticed that on Minikube Kafka had an ugly `CrashLoopBackOff` that crashes it around 3 times (roughly 80s) before the first Kafka broker runs successfully ( see [video](https://cl.ly/e015f4ceeefb)):

```bash
zk-zk-2   1/1   Running   0     17s
zk-zk-0   1/1   Running   0     19s
zk-zk-1   1/1   Running   0     21s
small-kafka-0   0/1   Pending   0     0s
small-kafka-0   0/1   Pending   0     0s
small-kafka-0   0/1   Pending   0     0s
small-kafka-0   0/1   ContainerCreating   0     0s
small-kafka-0   0/1   Running   0     19s
small-kafka-0   0/1   Error   0     26s
small-kafka-0   0/1   Running   1     27s
small-kafka-0   0/1   Error   1     36s
small-kafka-0   0/1   CrashLoopBackOff   1     48s
small-kafka-0   0/1   Running   2     49s
small-kafka-0   0/1   Error   2     57s
small-kafka-0   0/1   CrashLoopBackOff   2     68s
small-kafka-0   0/1   Running   3     82s
```
This is because of problems with PersistentVolumes provisioning, in particular the given issue stated in the Changelog above shows the actual error message: `Found directory /var/lib/kafka/lost+found, 'lost+found' is not in the form of topic-partition`. 

With setting `/var/lib/kafka/data` as the data directory this is fixed without the need of any data being deleted (as discussed in the issue).

### CODE REVIEW

**Review by gerred — APPROVED** [review]

File: repo/stable/kafka/versions/0/kafka-frameworkversion.yaml [hunk 1]
```diff
@@ -317,7 +312,7 @@ spec:
       strategy: serial
       phases:
         - name: deploy-kafka
-          strategy: parallel
+          strategy: serial
```
Comment (joerg84) [inline repo/stable/kafka/versions/0/kafka-frameworkversion.yaml:315]: Why this change?

**Review by joerg84 — APPROVED** [review]

File: repo/stable/kafka/versions/0/kafka-frameworkversion.yaml (hunk 1 above)
Comment (fabianbaier) [reply repo/stable/kafka/versions/0/kafka-frameworkversion.yaml:315]: We have it as `serial` here https://github.com/mesosphere/dcos-kafka-service/blob/master/frameworks/kafka/universe/config.json#L83 and here https://github.com/kudobuilder/kudo/blob/master/config/samples/kafka-frameworkversion.yaml#L315

I think this is the right step in light of kep-0006 and think this should be further discussed when looking at more details but for now I think this is the right approach.
